Formula/usbredir.rb

```diff
@@ -1,8 +1,8 @@
 class Usbredir < Formula
   desc "USB traffic redirection library"
   homepage "https://www.spice-space.org"
-  url "https://www.spice-space.org/download/usbredir/usbredir-0.8.0.tar.bz2"
-  sha256 "87bc9c5a81c982517a1bec70dc8d22e15ae197847643d58f20c0ced3c38c5e00"
+  url "https://www.spice-space.org/download/usbredir/usbredir-0.9.0.tar.xz"
+  sha256 "a3e167bf42bc7fe02c3c9db27d7767f1b8ce41b99ad14a4b0d0a60abe8bf56a6"
   license "GPL-2.0"
```

Comment:
```suggestion
  license all_of ["GPL-2.0-or-later", "LGPL-2.0-or-later"]
```
In the source tarball:
```
❯ rg -l 'license' | wc -l
17
❯ rg -l 'at your option' | wc -l
16
❯ rg -il 'lesser' | wc -l
12
❯ rg -il 'at your option' $(rg -il 'lesser') | wc -l
12
```